### ISSUE TITLE
vdir continually tries to add itself

### ISSUE THREAD

**Issue by EasyAsABC123** [opscode-cookbooks/iis#100]
```
Begin output of C:\Windows\System32\inetsrv\appcmd.exe add vdir /app.name:"Testfu Site/" /path:"/Content/Test" /physicalPath:"C:\wwwroot\shared" /logonMethod:ClearText /allowSubDirConfig:true ----
STDOUT: ERROR ( message:Failed to add duplicate collection element "/Content/Test". )
STDERR: 
```

**Comment by endakenny** [comment]
@EasyAsABC123  - just encountered this myself now - unfortunately its a show stopper for me. Looking forward to having a fix for this one.

**Comment by EasyAsABC123** [comment]
resolved #96

**Comment by EasyAsABC123** [comment]
@endakenny should be resolved in iis@3.0.8

**Comment by EasyAsABC123** [comment]
@endakenny did this resolve the issue for you as well?

**Comment by endakenny** [comment]
@EasyAsABC123  - yes this is resolved for me in current iis@3.0.9

**Comment by EasyAsABC123** [comment]
@endakenny sounds good, the official IIS is now IIS@3.0.10 and includes this fix.  Feel free to start pulling from the master branch https://github.com/opscode-cookbooks/iis as all fixes have been brought into it.
